src/System.ValueTuple/tests/System.ValueTuple.Tests.builds

```diff
@@ -2,8 +2,15 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.ValueTuple.Tests.csproj" />
     <Project Include="System.ValueTuple.Tests.csproj">
+      <TestTFMs>netcoreapp1.1</TestTFMs>
+    </Project>
+    <Project Include="System.ValueTuple.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcore50;net46</TestTFMs>
+    </Project>
```

Comment:
this is already included in the next element. why we need to add this one too? or we need to include this for Linux runs? if so, then having the listed TestTFMs looks wrong to me.